proxy/src/main/java/org/apache/rocketmq/proxy/service/message/LocalRemotingCommand.java

```diff
@@ -32,6 +32,7 @@ public static LocalRemotingCommand createRequestCommand(int code, CommandCustomH
         cmd.writeCustomHeader(customHeader);
         cmd.setExtFields(new HashMap<>());
         setCmdVersion(cmd);
+        cmd.makeCustomHeaderToNet();
```

Comment:
seems that this is irrelevant to the commit

Reply:
I saw your commit in another pr, maybe you can split them

Reply:
Include it to make the Test CI Pipeline pass